archivers/gnutar/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem 1.0
 
 name                gnutar
-version             1.29
+version             1.30
 revision            1
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.